### PR TITLE
chore(K8s): add Kubernetes usage instructions

### LINKED ISSUE #81: Kubernetes Integration
## Summary

This projects seems to supply the best DNS server in a container that one could ask for (especially when it comes to security features). Therefore, I'd like to help with documenting Kubernetes. I came across this as [coreDNS](https://coredns.io/) does not seem to be able to do DNSSEC validation (which I find to be ridiculous). Therefore, I can point coreDNS to this container in my cluster and let this container do the resolution over TLS/HTTP with DNSSEC etc.

## What would this help look like?

I am going to write a `Deployment`, `Service` and `configMap` which should suffice to get this up and running in a cluster. I would then add a document, maybe in the form of a `KUBERNETES.md`, because Kubernetes support should stay community-maintained (as it is a lot of work and maintainers most likely do not have the time).

## Why this issue?

I want to ask first before doing the work in order to avoid unnecessary work. If maintainers are fine with this, I will submit a PR in the future.

### PR DESCRIPTION
**Pull request by georglauterbach**
This adds the Kubernetes usage instructions mentioned in #81.

Closes #81

### CODE REVIEW

File: KUBERNETES.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Using `qmcgaw/dns` as the DNS Resolver in K8s
+
+This guide provides an introduction on how to use `qmcgaw/dns` as the DNS resolver in your Kubernetes cluster. This brings the advantage of supporting DoT, DoH, DNSSEC, rebinding protection and more without needing to heavily adjust already present components in your cluster. Note that we are already using `v2` (which is still in beta). You may use `v1´ as well, but make sure to change the environment variables' names and the image tag.
```
Comment (qdm12) [inline KUBERNETES.md:3]: Typo I think

```suggestion
This guide provides an introduction on how to use `qmcgaw/dns` as the DNS resolver in your Kubernetes cluster. This brings the advantage of supporting DoT, DoH, DNSSEC, rebinding protection and more without needing to heavily adjust already present components in your cluster. Note that we are already using `v2` (which is still in beta). You may use `v1` as well, but make sure to change the environment variables' names and the image tag.
```

**Review by qdm12 — APPROVED** [review]
Nice thanks!!

Maybe add a reference to that document in the Readme? (also ideally use absolute URLs so it works on Docker Hub too)

**Comment by georglauterbach** [comment]
> Nice thanks!!
> 
> Maybe add a reference to that document in the Readme? (also ideally use absolute URLs so it works on Docker Hub too)

I can do that of course :)

**Comment by georglauterbach** [comment]
@qdm12 Done. There is now a link (that works for the beta branch).

**Review by qdm12 — APPROVED** [review]

**Pull request merged**